resources/definitions/cartesio.def.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "cartesio",
+    "name": "Cartesio",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Scheepers",
+        "manufacturer": "Cartesio bv",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "has_machine_materials": true,
```

Comment:
In that case, you might want to simply disable "has_materials" for now. You could then update the definition once the materials are actually available. I think that would be better than having an empty material selector.